### PR TITLE
59 conda installation

### PR DESCRIPTION
**Pull request by bregeon**
Issue #59 

## What changes were proposed in this pull request?

- add `environment.yml` for conda installation, and updated `README.md`

How is the issue this PR is referenced against solved with this PR?

- issue is solved, although to be complete, one should also upload the package on `conda-forge`

## How was this patch tested?

- I tested only the installation not running anything... someone knowledgeable should do that.

### CODE REVIEW

File: environment.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+name: fink-client
+channels:
+  - defaults
+  - conda-forge
+dependencies:
+  - python>=3.9
+  - coverage>=4.2
+  - coveralls
+  - codecov
+  - python-confluent-kafka=1.9.2
```
Comment (JulienPeloton) [inline environment.yml:10]: Following #162 , this package should be upgraded to
```
confluent-kafka==2.0.2
```
to allow with python 3.11

**Review by JulienPeloton — APPROVED** [review]
Thanks @bregeon ! I just  left a minor comment before merging

**Comment by JulienPeloton** [comment]
I will upload the package on conda as soon as I get a bit of time :-)

File: environment.yml (hunk 1 above)
Comment (bregeon) [reply environment.yml:10]: `confluent-kafka=2.0.2` is not available on conda forge yet, [see](https://anaconda.org/conda-forge/python-confluent-kafka)
Let's wait a week or so :)

Comment (bregeon) [reply environment.yml:10]: or may be I could actually use `python-confluent-kafka>=1.9.2` so that conda can handle the different versions by itself?

Comment (bregeon) [reply environment.yml:10]: I have tried with `python-confluent-kafka>=1.9.2` and 
actually `confluent-kafka=2.0.2` is install when running the `pip install fink-client`.
So I think it's all fine also for the future.
I am going to commit that.

Comment (JulienPeloton) [reply environment.yml]: great, thanks!

**Pull request merged**